Automattic/sensei
# Remove image attribute from the object returned by map_task

Fixes #5931

### Changes proposed in this Pull Request

* Remove the "image" attributes from the tasks returned by the endpoint API related to Sensei Home;

### Testing instructions

On a WP installation containing Sensei LMS on this branch:

1. Run `curl --user "user:application password" http://your-host/wp-json/sensei-internal/v1/home | jq .tasks`;
2. Make sure that there are no `image` attributes in the output of the command above;
